bcdevexchange/wwwroot/css/style.css

```diff
@@ -174,6 +171,40 @@ h3 {
   font-weight: 500;
   padding: 8px 10px;
 }
+
+/*---- MW: I added styles here for 'hero's to be used across pages / To replace 'banner' styles -----*/

```

Comment:
Comments should probably not be explaining.  Something along the lines of:
/*------------ Begin Hero styling (replacing banner styles) -----------*/

/*-------------End Hero styling -----------------*/